kotlin-eclipse-ui/src/org/jetbrains/kotlin/ui/editors/annotations/AnnotationManager.kt

```diff
@@ -82,6 +82,8 @@ public object AnnotationManager {
             setAttribute(IMarker.SEVERITY, annotation.markerSeverity)
             setAttribute(IMarker.CHAR_START, annotation.offset)
             setAttribute(IMarker.CHAR_END, annotation.endOffset)
+			setAttribute(IMarker.LOCATION, "line " + (annotation.line + 1))
+			setAttribute(IMarker.LINE_NUMBER, annotation.line + 1)
```

Comment:
I propose to keep in `annotation` object already incremented line number to avoid writing `+ 1`. Then you also be able to write here `"line ${annotation.line}"`